.github/workflows/linux.yml

```diff
@@ -51,12 +57,20 @@ jobs:
           -name "GitHub"
           -username "BlueQuartzSoftware"
           -password "${{secrets.GITHUB_TOKEN}}"
+      - name: Download Test Files
+        shell: bash
+        run: >
+          wget --quiet "https://github.com/dream3d/DREAM3D_Data/releases/download/v6_7/TestFiles_6_7.tar.gz" -O ${{github.workspace}}/DREAM3D_Data/TestFiles.tar.gz
```

Comment:
Is the data in this zip not already in DREAM3D_Data?

Reply:
It is NOT in DREAM3D_Data. The data is getting large and Git is asking to use Git LFS instead. Using this approach it is easier to just version the data used, IMO anyways. Maybe at some point in the near future we can just post the data to our own "Data Repository" that has enough bandwidth to serve all  the build bots.